airflow/hooks/base_hook.py

```diff
@@ -80,7 +80,7 @@ def get_connection(cls, conn_id):
         conn = random.choice(cls.get_connections(conn_id))
         if conn.host:
             log = LoggingMixin().log
-            log.info("Using connection to: %s", conn.debug_info())
+            log.debug("Using connection to: %s", conn.debug_info())
```

Comment:
I can revert it. I changed it because when I was running my DAG this was polluting my log, displaying information about parsed DAGs that I wasn't running at the moment.
Depends if we want the logs to be super clean or not.

Reply:
I'd propose it as a very small separate change and see if it will be accepted ;). This way even if it is merged and someone complains, only this change could be reverted easily.

Reply:
Ok, I'll withdraw it from this PR.